sidecar/internal/service/reclaimspace.go

```diff
@@ -35,15 +40,17 @@ type ReclaimSpaceServer struct {
 	controllerClient csiReclaimSpace.ReclaimSpaceControllerClient
 	nodeClient       csiReclaimSpace.ReclaimSpaceNodeClient
 	kubeClient       *kubernetes.Clientset
+	stagingPath      *string
```

Comment:
why a `*string`? using a non-pointer `string` will be simpler.

Reply:
the flag.string returns *string

Reply:
that is no an excuse to use a pointer in this struct :smiley:

Reply:
I will change it.